AnnotationModule.py

```diff
@@ -0,0 +1,63 @@
+import csv
+import xml.etree.ElementTree as ET
+import cv2
+import numpy as np
+import openslide
+import os
+from skimage import io
+from BaseImage import printMaskHelper
+from PIL import Image, ImageDraw
+
+def getPoints(xmlfile):
+    # create element tree object
+    tree = ET.parse(xmlfile)
+
+    # get root element
+    root = tree.getroot()
+    annCount = 0
+
+    # create empty list for annotations
+    annotations = []
+
+    for annotation in root.findall('Annotation'):
+        for regions in annotation.findall('Regions'):
+            for region in regions.findall('Region'):
+                for vertices in region.findall('Vertices'):
+                    points[annCount] = [None] * len(vertices.findall('Vertex'))
+                    k = 0
+                    for vertex in vertices.findall('Vertex'):

```

Comment:
can you use "enumerate" here instead of the k counter?

Reply:
Sure, as soon as I learn how to use enumerate.